rskj-core/src/main/java/org/ethereum/core/genesis/GenesisLoaderImpl.java

```diff
@@ -230,6 +232,20 @@ private void setupPrecompiledContractsStorage(Repository repository) {
         }
     }
 
+    public static InputStream loadGenesisFile(String fileName) {
+        InputStream inputStream = GenesisLoaderImpl.class.getResourceAsStream("/genesis/" + fileName);
+        if (inputStream != null) {
+            return inputStream;
+        }
+
+        try {
+            return Files.newInputStream(Paths.get(fileName));
+        } catch (Exception e) {
+            logger.error("Could not load file due to:", e);
+            return null;
```

Comment:
Why return null here if at the end, you will throw an exception on the method  `readGenesisJson`? Wdyt in throw the exception here straight away?